apps/site/src/app/(main)/volunteer/page.tsx

```diff
@@ -0,0 +1,9 @@
+import { Metadata } from "next";
+
+export const revalidate = 60;
```

Comment:
Thought: not sure if specifying a revalidation time is necessary since the page content doesn't fetch dynamic data.